capstone/target/step-96-2020-1/WEB-INF/appengine-web_copy.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It seems we already have appengine-web.xml. Why do we need a copy here?

Reply:
The whole maven folder seems to have duplicates and Maggie is looking into it. Will leave this here for now.